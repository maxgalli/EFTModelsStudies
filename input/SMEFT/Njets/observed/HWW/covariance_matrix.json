{
    "r_Njets_0": {
        "r_Njets_0": 0.018102216430889786,
        "r_Njets_1": 0.002964218329062164,
        "r_Njets_2": 0.011276879835489516,
        "r_Njets_3": 0.00729904417997148,
        "r_Njets_4": 0.025339505849128183
    },
    "r_Njets_1": {
        "r_Njets_0": 0.002964218329062164,
        "r_Njets_1": 0.04535922431321743,
        "r_Njets_2": -0.012444269875341847,
        "r_Njets_3": 0.04238048353191041,
        "r_Njets_4": 0.022515264422011422
    },
    "r_Njets_2": {
        "r_Njets_0": 0.011276879835489516,
        "r_Njets_1": -0.012444269875341847,
        "r_Njets_2": 0.17902697355907024,
        "r_Njets_3": -0.20411527955368483,
        "r_Njets_4": 0.17247342118534378
    },
    "r_Njets_3": {
        "r_Njets_0": 0.00729904417997148,
        "r_Njets_1": 0.04238048353191041,
        "r_Njets_2": -0.20411527955368483,
        "r_Njets_3": 1.7559450425394028,
        "r_Njets_4": -0.6883344389799325
    },
    "r_Njets_4": {
        "r_Njets_0": 0.025339505849128183,
        "r_Njets_1": 0.022515264422011422,
        "r_Njets_2": 0.17247342118534378,
        "r_Njets_3": -0.6883344389799325,
        "r_Njets_4": 3.788311387724089
    }
}