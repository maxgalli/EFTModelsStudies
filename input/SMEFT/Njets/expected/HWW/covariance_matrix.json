{
    "r_Njets_0": {
        "r_Njets_0": 0.01649299766138375,
        "r_Njets_1": -0.0004775125930284095,
        "r_Njets_2": 0.0010266875075112725,
        "r_Njets_3": -0.0007630647450406647,
        "r_Njets_4": 0.000427644821591133
    },
    "r_Njets_1": {
        "r_Njets_0": -0.0004775125930284095,
        "r_Njets_1": 0.03528260656774747,
        "r_Njets_2": -0.017404012838307246,
        "r_Njets_3": 0.016668915038686536,
        "r_Njets_4": 3.229324343378642e-05
    },
    "r_Njets_2": {
        "r_Njets_0": 0.0010266875075112725,
        "r_Njets_1": -0.017404012838307246,
        "r_Njets_2": 0.12199765266025481,
        "r_Njets_3": -0.13477486453048038,
        "r_Njets_4": 0.05469344205994052
    },
    "r_Njets_3": {
        "r_Njets_0": -0.0007630647450406647,
        "r_Njets_1": 0.016668915038686536,
        "r_Njets_2": -0.13477486453048038,
        "r_Njets_3": 1.0877051600719663,
        "r_Njets_4": -0.5083033890837301
    },
    "r_Njets_4": {
        "r_Njets_0": 0.000427644821591133,
        "r_Njets_1": 3.229324343378642e-05,
        "r_Njets_2": 0.05469344205994052,
        "r_Njets_3": -0.5083033890837301,
        "r_Njets_4": 2.470192543612844
    }
}